packages/shared/components/Profile.svelte

```diff
@@ -1,29 +1,24 @@
 <script lang="ts">
-    import { DeveloperIndicatorPill, Icon, NetworkIconBadge, StrongholdBadge, Text, TextType } from 'shared/components'
-
+    import { IPersistedProfile } from '@core/profile'
     import { getInitials as _getInitials } from '@core/utils'
-    import { NetworkProtocol, NetworkType } from '@core/network'
+    import { DeveloperIndicatorPill, Icon, NetworkIconBadge, StrongholdBadge, Text, TextType } from 'shared/components'
 
-    export let name: string = ''
-    export let id: string = ''
-    export let networkType: NetworkType = undefined
-    export let networkProtocol: NetworkProtocol = undefined
+    export let profile: IPersistedProfile
     export let classes: string = undefined
     export let bgColor: string = ''
-    export let isDeveloper: boolean = false
     export let isLedgerProfile: boolean = false
     export let updateRequired: boolean = false
```

Comment:
At least `isLedgerProfile` can be determined from the profile data passed in, I suppose we don't want to couple the `updateRequired` logic to this component though so best to leave that in the parent :thinking: